client_test.go

```diff
@@ -1,18 +1,387 @@
 package makroud_test
 
 import (
+	"context"
+	"database/sql"
 	"testing"
+	"time"
 
+	"github.com/lib/pq"
 	"github.com/stretchr/testify/require"
+	"github.com/ulule/loukoum/v3"
 
 	"github.com/ulule/makroud"
 )
 
-func TestPing(t *testing.T) {
+func TestClient_New(t *testing.T) {
+	is := require.New(t)
+
+	driver, err := makroud.New(Options()...)
+	is.NoError(err)
+	is.NotEmpty(driver)
+
+	err = driver.Ping()
```

Comment:
```suggestion
	is.NoError(driver.Ping())
```

Reply:
I prefer to keep the `given / when / then` pattern here.